src/tests/fixtures/projects/dummy-project.yaml

```diff
@@ -378,7 +380,18 @@ project-with-fastly-gcs:
                 bucket: my-bucket
                 path: my-project/
                 period: 1800
-                    
+
+project-with-fastly-shielding:
+    repo: ssh://git@github.com/elifesciences/dummy3
+    subdomain: www
+    aws:
+        fastly:
+            subdomains:
+                - "{instance}--cdn-of-www"
+            subdomains-without-dns:
+                - "{instance}--www"
```

Comment:
Currently we'll have to set this (as the `Host` header is changed). Could make it automatic?

Reply:
Yes but as a follow-up